src/lib/products.ts

```diff
@@ -18,6 +18,22 @@ const productSlugsToNames: { [slug in ProductSlug]: ProductName } = {
 	vagrant: 'Vagrant',
 }
 
+/**
+ * A map of product slugs to their "dot io" site hostname.
+ */
+const productSlugsToHostNames: { [slug in ProductSlug]: string } = {
+	boundary: 'boundaryproject.io',
+	consul: 'consul.io',
+	hcp: 'cloud.hashicorp.com',
+	nomad: 'nomadproject.io',
+	packer: 'packer.io',
+	sentinel: 'docs.hashicorp.com',
+	terraform: 'terraform.io',
+	vagrant: 'vagrantup.com',
+	vault: 'vaultproject.io',
+	waypoint: 'waypointproject.io',
+}
```

Comment:
💭 this is duplicated in `src/components/opt-in-out/helpers/platform-options.ts` and `src/lib/remark-plugins/rewrite-tutorial-links/index.ts`. What do you think about consolidating?

Reply:
`src/lib/remark-plugins/rewrite-tutorial-links/index.ts` is already updated in this PR, but I didn't see that in `src/components/opt-in-out/helpers/platform-options.ts` too! I'll update that file to reference this new bit as well 😊

Reply:
@BRKalow Updated in https://github.com/hashicorp/dev-portal/pull/768/commits/8bada726d340bf3b348c7321b9c691b3e64152af! ✅